package.json

```diff
@@ -167,6 +166,7 @@
 		"git-revision-webpack-plugin": "^3.0.6",
 		"hard-source-webpack-plugin-fixed-hashbug": "^0.13.2",
 		"html-minifier": "^4.0.0",
+		"husky": "^5.2.0",
```

Comment:
did you run npm i again after moving this to update package-lock

Reply:
Whoops! Will do